src/components/motion/MotionController.cpp

```diff
@@ -39,6 +39,17 @@ bool MotionController::ShouldWakeUp(bool isSleeping) {
   }
   return false;
 }
+bool MotionController::ShouldSleep(bool isSleeping) {
+  bool ret = false;
+
+  if (y >= lastYForSleep + 192 && !isSleeping)
+    ret = true;
+
```

Comment:
isSleeping doesn't need to be passed to the function, it could just be checked in systemtask. Also missing braces

Reply:
In that case, couldn't isSleeping also just be checked in systemtask for raise to wake?

Reply:
Yes it could, and looks like you fixed it in the other PR 👍.